client/realm_management.go

```diff
@@ -118,8 +119,10 @@ type InstallInterfaceRequest struct {
 }
 
 // InstallInterface builds a request to install a new major version of an Interface into the Realm.
-func (c *Client) InstallInterface(realm string, interfacePayload interfaces.AstarteInterface) (AstarteRequest, error) {
+func (c *Client) InstallInterface(realm string, interfacePayload interfaces.AstarteInterface, isAsync bool) (AstarteRequest, error) {
 	callURL := makeURL(c.realmManagementURL, "/v1/%s/interfaces", realm)
+	callURL = setupURLQuery(callURL, map[string]string{"async_operation": strconv.FormatBool(isAsync)})
```

Comment:
Defining the map one row above would increase code readability. Can you do that?

Reply:
Yes!

Reply:
Updated